api/src/test/resources/io/kaoto/backend/api/resource/amq-amq-multi.json

```diff
@@ -0,0 +1,2798 @@
+[
```

Comment:
JSON responses now are arrays/lists, not single objects. So I had to copy some resources files and added the `multi` string to distinguish.